src/plugins/Core/plugins/etm/registry/registry.py

```diff
@@ -0,0 +1,88 @@
+import typing
+
+T1 = typing.TypeVar("T1")
+
+
+class ResourceLocation:
+
+    namespace: str
+    path: str
+
+    def __init__(self, namespace: str, path: str):
+        self.namespace = namespace
+        self.path = path
+
+    def getNamespace(self) -> str:
+        return self.namespace
+
+    def getPath(self) -> str:
+        return self.namespace
```

Comment:
```suggestion
        return self.path
```

这里不应该是 `self.path` 吗